design/compile-commands.md

```diff
@@ -100,6 +100,9 @@ compile commands for them.
 If no command is available for a header, but a file has been opened that
 transitively includes it, then that file's command will be used. To enable this
 [TUScheduler] keeps a `HeaderIncluderCache` to look up the relevant file.
+[This issue](https://github.com/clangd/clangd/issues/123) tracks a further
```

Comment:
I was talking about the delta here. We should update the docs once we have some inference through Index, but I don't think it is useful to talk about this intermediate state here.

Reply:
Thanks, removed this change.